### PR TITLE
dont do async: false

### LINKED ISSUE #684: pulling in topics from autocomplete uses deprecated synchronous ajax call
screen shot 2016-09-28 at 8 48 46 pm

make this happen async instead

### PR DESCRIPTION
**Pull request by Connoropolous**
remove fetchContained wasn't doing anything anyways

fixes #684

### CODE REVIEW

File: frontend/src/Metamaps/AutoLayout.js [hunk 1]
```diff
@@ -59,8 +59,7 @@ const AutoLayout = {
       }
     }
   },
-  coordsTaken: function (x, y, map) {
-    const mappings = map.getMappings()
+  coordsTaken: function (x, y, mappings) {
```
Comment (devvmh) [inline frontend/src/Metamaps/AutoLayout.js]: nice

File: frontend/src/Metamaps/Topic.js [hunk 2]
```diff
@@ -35,30 +35,14 @@ const Topic = {
...
-      }
-    }
+      $.ajax({
+        url: '/topics/' + id + '.json',
+        success: function (data) {
+          Metamaps.Topics.add(data)
+          callback(Metamaps.Topics.get(id))
+        }
+      })
+    } else callback(Metamaps.Topics.get(id))
```
Comment (devvmh) [inline frontend/src/Metamaps/Topic.js]: what if callback is null?

Comment (Connoropolous) [inline frontend/src/Metamaps/Topic.js]: should I catch that case then?

Comment (devvmh) [inline frontend/src/Metamaps/Topic.js]: i guess we could do

`const identityFunction = x => x`
`callback = identifyFunction`

Comment (devvmh) [inline frontend/src/Metamaps/Topic.js]: (as a default argument)

**Pull request merged**